### DISCUSSION TITLE
Restrict export rights

### DISCUSSION THREAD

**Discussion by gmestdacOLV** [directus/directus#19392] [Feature Requests]
### Summary

It would be nice to have the option to prevent users from exporting data.
I'm aware the user could still extract the data they have access to "manually", but ensuring they can't obtain all data at the click of a button would suffice for many purposes. 
I see 3 potential levels where this could be implemented:
- have the option to remove/disable the export functionality entirely at the project level
- allow for restrictions at the table level
- allow for restuctions at the row and collumn level

The first option would in fact already be sufficient for my usecase.

### Motivation

We want to use directus in a medical context as an alternative to excel. Users branching off from the tables managed in directus by editing an export to excel is a real concern. 

### Detailed Design

None, not a complex feature

### Must Have:
- Disabling exports project-wide 

### Should Have:
- Table level export restrictions

### Could Have:
- More fine-grained export restrictions

### Won't Have:
- /

### Drawbacks

None

### Alternatives

Perhaps some workaround already exists that would make it possible to prevent exports, for example by configuring some aspect related exporting wrong on purpose. This would also suffice for me, as I would like to prevent exports in general.

### Adoption Strategy

I see no reason why any of the options would be a breaking change

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
